service/csv/csv.go

```diff
@@ -0,0 +1,93 @@
+package csv
+
+import (
+	"encoding/csv"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/rodrigomkd/go-rest-api/model"
+)
+
+type ICSVService interface {
+	ReadCSV(path string) ([][]string, error)
+	SaveCSV(path string, records [][]string)
+	SaveActivities(path string, activities []model.Activity)
+}
+
+type CSVService struct {
```

Comment:
I would recommend having the file path as an attribute for this struct instead of receiving it in as a function param.